luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -88,11 +111,11 @@ def gnomad_svs_af(self):
 
     @row_annotation(name='gnomad_svs_AC')
     def gnomad_svs_ac(self):
-        return unsafe_cast_int32(self.mt.info.gnomad_svs_AC)
+        return unsafe_cast_int32(self.mt.info.gnomAD_V2_AC)
 
     @row_annotation(name='gnomad_svs_AN')
     def gnomad_svs_an(self):
-        return unsafe_cast_int32(self.mt.info.gnomad_svs_AN)
+        return unsafe_cast_int32(self.mt.info.gnomAD_V2_AN)
```

Comment:
The `gnomAD_V2_AC` and `gnomAD_V2_AN` use to be `gnomAD_V2_AC_AF` and `gnomAD_V2_AN_AF`. Did they change the field names?

Reply:
Yes.  I fixed the filenames on a separate pr... then accidentally typoed them to `gnomad_svs_AN`.... `gnomAD_V2_AN` is correct here.